### ISSUE TITLE
Ready for review

### ISSUE THREAD

**Issue by amyjoui** [uidaholib/collectionbuilder-cdm-template#91]
They look as complete to me as far as my knowledge of CollectionBuilder goes. :)
- [ ] Lewiston Orchards Life (lewistonorch). I had troubles with the carousel on the home page (Sample Items) but I think Michelle figured it out. I took out the Map, cause its only one location, but added subjects based on AAT. 
- [ ] C.H. Shattuck Lantern Slides (shattuck). I kept the locations but didn't include a Map because of the ambiguity of the locations. I also didn't include subjects (this also wasn't in the original), but if you think it will be beneficial I can. 
Thanks!
@dcnb @evanwill @owikle

**Comment by dcnb** [comment]
Hi @amyjoui 

Finally started to get to this. 

A couple notes: 

### Sections
- There's a new menu at the top of about pages. It creates links to various sections you want linked. Basically anything that's a h2, starting with "## "
- First step with this is to add sections. I added an "About CH Shattuck" and "About the Collection" section to the shattuck collection, for instance. Basically, you already had sections, they just weren't labeled. 

### Facets
- for the `metadata-facets-fields` section of the "Data" section of the theme.yml, make sure and add any idiosyncratic fields there. 
- I use those to determine what we should put in for the subject clouds etc but I also like making anything unique to the collection exposed to indexing and stuff like that in a group. I don't know how effective this will be in the long run, but if you wouldn't mind indulging me, please spend a bit of time on this section. 

Everything looks great!  I will have it all up today!

**Comment by amyjoui** [comment]
Hi Devin,

I'm sure these emails will go to you, but since at lease one is a 'no reply' i'm Cc'ing you directly too. Sorry for any redundancy!

Thanks for looking these over!

Sections:
Yes, that makes perfect sense. I was also wondering what all I should include in the About section. So like do I keep the 'About CollectionBuilder' and technical portions?

Facets:
Yes, of course. I found the metadata-facets-fields part in the second collection I did (Shattuck), and changed it to include 'locations' rather than 'subjects' cause there were no subjects, but that was as far as I went with that one. I will definitely look out for any other fields to include.

I know I'm slow at it, but its been fun to learn and its nice to be able to show I've accomplished something.

Thanks!
amy
________________________________
